libs/challenge-registry/user-profile/src/lib/user-profile-stats/user-profile-stats.component.html

```diff
@@ -8,9 +8,9 @@ <h4>1</h4>
     <p class="mat-caption">stars</p>
   </div>
   <div class="stat-item action-btn" *ngIf="loggedIn">
-    <mat-icon aria-hidden="true" class="card-icon">edit</mat-icon> Edit Profile
+    <mat-icon aria-hidden="true" class="stat-card-icon">edit</mat-icon> Edit Profile
```

Comment:
@vpchung  just a note why it's been changed here: Both `challenges-card` and  `user-profile-starred` component have `card-icon` class, which results in the color of `card-icon` will be overwritten by what defined in `challenge-card` (It might be one caveat from using "less" descriptive class name).

<img width="322" alt="Screen Shot 2022-09-30 at 8 06 24 PM" src="https://user-images.githubusercontent.com/73901500/193372190-0aa14c35-5ea1-40b1-884c-52dd439797ef.png">

Reply:
@rrchai yeah, I realize that when I made the first PR, hence why I think we should eventually extract common classes, then do specific defs!  But... one thing at a time 🙃